2022-11-15 16:53:04,171 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Configure stats pid to 1637445
2022-11-15 16:53:04,171 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from /nfs/homedirs/zhz/.config/wandb/settings
2022-11-15 16:53:04,173 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/settings
2022-11-15 16:53:04,173 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-11-15 16:53:04,175 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-11-15 16:53:04,175 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Applying login settings: {'api_key': '***REDACTED***'}
2022-11-15 16:53:04,177 INFO    MainThread:1637445 [wandb_init.py:_log_setup():476] Logging user logs to /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/run-20221115_165304-2ga9meiw/logs/debug.log
2022-11-15 16:53:04,179 INFO    MainThread:1637445 [wandb_init.py:_log_setup():477] Logging internal logs to /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/run-20221115_165304-2ga9meiw/logs/debug-internal.log
2022-11-15 16:53:04,181 INFO    MainThread:1637445 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f16c84fccc0>
2022-11-15 16:53:04,183 INFO    MainThread:1637445 [wandb_init.py:init():516] calling init triggers
2022-11-15 16:53:04,184 INFO    MainThread:1637445 [wandb_init.py:init():520] wandb.init called with sweep_config: {}
config: {}
2022-11-15 16:53:04,186 INFO    MainThread:1637445 [wandb_init.py:init():536] re-initializing run, found existing run on stack: 1r55bynf
2022-11-15 16:53:04,191 INFO    MainThread:1637445 [wandb_run.py:_finish():1746] finishing run uncertainty_tum/test-project/1r55bynf
2022-11-15 16:53:04,191 INFO    MainThread:1637445 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-11-15 16:53:04,193 INFO    MainThread:1637445 [wandb_run.py:_restore():1932] restore
2022-11-15 16:53:04,195 INFO    MainThread:1637445 [wandb_run.py:_restore():1938] restore done
2022-11-15 16:53:10,791 INFO    MainThread:1637445 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-11-15 16:53:10,792 INFO    MainThread:1637445 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2022-11-15 16:53:10,800 INFO    MainThread:1637445 [wandb_run.py:_footer_sync_info():3333] logging synced files
2022-11-15 16:53:10,809 INFO    MainThread:1637445 [wandb_init.py:init():569] starting backend
2022-11-15 16:53:10,809 INFO    MainThread:1637445 [wandb_init.py:init():573] setting up manager
2022-11-15 16:53:10,817 INFO    MainThread:1637445 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-11-15 16:53:10,822 INFO    MainThread:1637445 [wandb_init.py:init():580] backend started and connected
2022-11-15 16:53:10,848 INFO    MainThread:1637445 [wandb_run.py:_label_probe_notebook():1113] probe notebook
2022-11-15 16:53:10,848 INFO    MainThread:1637445 [wandb_run.py:_label_probe_notebook():1123] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-11-15 16:53:10,850 INFO    MainThread:1637445 [wandb_init.py:init():658] updated telemetry
2022-11-15 16:53:10,918 INFO    MainThread:1637445 [wandb_init.py:init():694] communicating run to backend with 60 second timeout
2022-11-15 16:53:12,270 INFO    MainThread:1637445 [wandb_run.py:_on_init():2000] communicating current version
2022-11-15 16:53:12,394 INFO    MainThread:1637445 [wandb_run.py:_on_init():2004] got version response 
2022-11-15 16:53:12,394 INFO    MainThread:1637445 [wandb_init.py:init():728] starting run threads in backend
2022-11-15 16:53:16,864 INFO    MainThread:1637445 [wandb_run.py:_console_start():1980] atexit reg
2022-11-15 16:53:16,864 INFO    MainThread:1637445 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.WRAP_RAW
2022-11-15 16:53:16,864 INFO    MainThread:1637445 [wandb_run.py:_redirect():1903] Wrapping output streams.
2022-11-15 16:53:16,864 INFO    MainThread:1637445 [wandb_run.py:_redirect():1925] Redirects installed.
2022-11-15 16:53:16,865 INFO    MainThread:1637445 [wandb_init.py:init():765] run started, returning control to user process
2022-11-15 16:53:17,368 INFO    MainThread:1637445 [wandb_init.py:_pause_backend():389] pausing backend
2022-11-15 16:53:17,369 INFO    MainThread:1637445 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-11-15 16:57:10,492 INFO    MainThread:1637445 [wandb_init.py:_resume_backend():398] resuming backend
2022-11-15 16:57:10,561 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Configure stats pid to 1637445
2022-11-15 16:57:10,561 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from /nfs/homedirs/zhz/.config/wandb/settings
2022-11-15 16:57:10,563 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/settings
2022-11-15 16:57:10,565 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-11-15 16:57:10,565 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-11-15 16:57:10,565 INFO    MainThread:1637445 [wandb_setup.py:_flush():68] Applying login settings: {'api_key': '***REDACTED***'}
2022-11-15 16:57:10,567 INFO    MainThread:1637445 [wandb_init.py:_log_setup():476] Logging user logs to /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/run-20221115_165710-2dl7103k/logs/debug.log
2022-11-15 16:57:10,569 INFO    MainThread:1637445 [wandb_init.py:_log_setup():477] Logging internal logs to /nfs/homedirs/zhz/uncertainty-quantification/src/posterior_networks/wandb/run-20221115_165710-2dl7103k/logs/debug-internal.log
2022-11-15 16:57:10,571 INFO    MainThread:1637445 [wandb_init.py:init():516] calling init triggers
2022-11-15 16:57:10,572 INFO    MainThread:1637445 [wandb_init.py:init():520] wandb.init called with sweep_config: {}
config: {}
2022-11-15 16:57:10,573 INFO    MainThread:1637445 [wandb_init.py:init():536] re-initializing run, found existing run on stack: 2ga9meiw
2022-11-15 16:57:10,577 INFO    MainThread:1637445 [wandb_run.py:_finish():1746] finishing run uncertainty_tum/test-project/2ga9meiw
2022-11-15 16:57:10,578 INFO    MainThread:1637445 [jupyter.py:save_history():448] not saving jupyter history
2022-11-15 16:57:10,579 INFO    MainThread:1637445 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-11-15 16:57:10,579 INFO    MainThread:1637445 [wandb_init.py:_jupyter_teardown():408] cleaning up jupyter logic
2022-11-15 16:57:10,581 INFO    MainThread:1637445 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-11-15 16:57:10,582 INFO    MainThread:1637445 [wandb_run.py:_restore():1932] restore
2022-11-15 16:57:10,582 INFO    MainThread:1637445 [wandb_run.py:_restore():1938] restore done
2022-11-15 16:57:14,536 INFO    MainThread:1637445 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-11-15 16:57:14,536 INFO    MainThread:1637445 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2022-11-15 16:57:14,544 INFO    MainThread:1637445 [wandb_run.py:_footer_sync_info():3333] logging synced files
